lib/ansible/module_utils/crypto.py

```diff
@@ -1883,3 +1884,22 @@ def quick_is_not_prime(n):
     # TODO: maybe do some iterations of Miller-Rabin to increase confidence
     # (https://en.wikipedia.org/wiki/Miller%E2%80%93Rabin_primality_test)
     return False
+
+
+python_version = (sys.version_info[0], sys.version_info[1])
+if python_version >= (2, 6) or python_version >= (3, 1):
+    # Ansible still supports Python 2.6 on remote nodes
```

Comment:
If this works on 2.6 (`>=`, not `>`), why the alternative implementation?

Reply:
That should have been 2.7... Thanks for spotting :)